content/ja/tightly-coupled-architectures.md

```diff
@@ -0,0 +1,18 @@
+---
+title: 密結合アーキテクチャ
+status: 完了
+category: プロパティ
+tags: ["基礎", "アーキテクチャ", "プロパティ"]
+---
+
+密結合アーキテクチャは、アプリケーション構成要素の多くが互いに依存しているようなアーキテクチャを指します
+(その逆は[疎結合アーキテクチャ](/loosely-coupled-architecture/)と呼ばれます)。
```

Comment:
please add ja path in link like below.
/ja/loosely-coupled-architecture/

Reply:
Thank you for your comments.
I fixed the link in [cf90e8c](https://github.com/cncf/glossary/pull/2365/commits/cf90e8c7ec15ac133c6defe0e2a9882e4863be09).